README.md

```diff
@@ -99,14 +102,16 @@ Select 1:
 
 What is the difference between a fork and a clone?
 
-```text
-Your answer here.
+```
+8. a clone pulls the contents of a parent repository directly on your computer and it is linked to the repo it came from. if you clone a master repo you do not have access to, you will not be able to make any changes to it. a fork makes a copy of a repository (usually the master) on git hub so that you can safely make modifications without compromising the master.
+
```

Comment:
:+1:  A clone is a version of someone else's repository downloaded to your computer. A fork is a copy of someone else's repository, not necessarily downloaded to your computer. You can clone a fork; you can't fork a clone.